static/js/index.js

```diff
@@ -0,0 +1,167 @@
+const details = [
+  {
+    id: 1,
+    title: 'Tonic',
+    list: {
+      list1: 'CANOPY',
+      list2: 'Back End Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/three.png',
+    desc: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    lang: {
+      lang1: 'html',
+      lang2: 'css',
+      lang3: 'js',
+      lang4: 'ruby',
+    },
+    category: 'project1',
+  },
+  {
+    id: 2,
+    title: 'Multi-Post Stories',
+    list: {
+      list1: 'FACEBOOK',
+      list2: 'Full Stack Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/four.png',
+    desc: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project2',
+  },
+  {
+    id: 3,
+    title: 'Facebook 360',
+    list: {
+      list1: 'FACEBOOK',
+      list2: 'Full Stack Dev',
+      list3: '2015',
+    },
+    image: 'static/imgs/Snapshoot Portfolio.png',
+    desc: 'Exploring the future of media in Facebooks first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project3',
+  },
+  {
+    id: 4,
+    title: 'Uber',
+    list: {
+      list1: 'Uber',
+      list2: 'Lead Developer',
+      list3: '2015',
+    },
+    image: 'static/imgs/two.png',
+    desc: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    lang: {
+      lang1: 'html',
+      lang2: 'Ruby',
+      lang3: 'css',
+      lang4: 'javaScript',
+    },
+    category: 'project4',
+  },
+];
+// eslint-disable-next-line no-plusplus
+
+// ==========================================================================
+const burger = document.querySelector(
+  '.header__container__parhead__nav__ul__responsive',
+);
+const iwq = document.querySelector('.iwq');
+const parpopup = document.querySelector('.parpopup');
+// eslint-disable-next-line camelcase
```

Comment:
- [ ] I noticed that you disabled **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1: